english/Rules to Success/Rules.txt

```diff
@@ -0,0 +1,17 @@
+Get comfortable being uncomfortable. Get outside your comfort zone on a regular basis. The more you do, the wider that area becomes. Things that used to seem impossible can become part of your everyday life.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.